### ISSUE TITLE
URI field is too strict

### ISSUE THREAD

**Issue by kecso** [webgme/webgme-taxonomy#499]
I believe the URI field should allow regular http based addresses as not sure how external elements should be referred otherwise (or the naming is confusing if it is intended only for UDCP internal links)

**Comment by brollb** [comment]
Yeah, currently the plan has been to add another storage adapter for http/s endpoints (which doesn't support upload/append).
